lang.cpp

```diff
@@ -100,45 +107,13 @@ int main(){
   }
 
 
-vector<vector<string> > vm_comands;
```

Comment:
It would be better to break the code into functional fragments, and not be confused to fit everything into one action. It is required to separate syntax, semantic analysis and code generation